src/.github/workflows/publish.yml

```diff
@@ -146,7 +121,7 @@ jobs:
             feat: update storybook site (auto)
 
   publish-pypi:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref, 'refs/tags/')
```

Comment:
We don't want nightly builds for pyhton packages?

Reply:
Indeed, we currently have no strategy for publishing non-release python packages.